pages/people/index.js

```diff
@@ -107,8 +107,7 @@ const HorizontalLettersMenu = ({ linkedLetters }) => {
 }
 
 /*
- * people are coming into this component from
- * fetchStrapiPeople with this shape:
+ * people are coming into this component with this shape:
```

Comment:
should we also filter the people array by the active field? currently it's pulling in both active and inactive staff members for the staff page

Reply:
ooh yes. thanks for catching that!